lib/workers.js

```diff
@@ -340,6 +355,7 @@ class Workers extends EventEmitter {
           this.emit(event.test.after, repackTest(message.data));
           break;
         case event.all.after:
+          failedTestRerun.writeFailedTests(failedTests);
```

Comment:
sorry, this looks like breaking the abstraction hierarchy. 
this class represents only workers and should know nothing about failedTestRerun